cmd/legacy_param_mapper.go

```diff
@@ -141,6 +141,7 @@ func PopulateNewConfigFromLegacyFlagsAndConfig(c cliContext, flags *flagStorage,
 	overrideWithFlag(c, "anonymous-access", &resolvedConfig.GcsAuth.AnonymousAccess, anonymousAccess)
 	overrideWithFlag(c, "kernel-list-cache-ttl-secs", &resolvedConfig.FileSystem.KernelListCacheTtlSecs, kernelListCacheTTLSecs)
 
+	cfg.OverrideWithLoggingFlags(resolvedConfig, resolvedConfig.Debug.Fuse, resolvedConfig.Debug.Gcs, resolvedConfig.Debug.LogMutex)
```

Comment:
IIUC, this should be in a separate method as we won't be using PopulateNewConfigFromLegacyFlagsAndConfig function post migration but this is needed on new configs regardless for backward compatibility.

Reply:
I've moved it to Rationalize.